tests/fail/intrinsics/simd-div-by-zero.rs

```diff
@@ -8,8 +8,10 @@ extern "platform-intrinsic" {
 #[allow(non_camel_case_types)]
 struct i32x2(i32, i32);
 
-fn main() { unsafe {
-    let x = i32x2(1, 1);
-    let y = i32x2(1, 0);
-    simd_div(x, y); //~ERROR Undefined Behavior: dividing by zero
-} }
+fn main() {
+    unsafe {
```

Comment:
Mirroring comment from https://github.com/rust-lang/miri/pull/2097#discussion_r862436832:

> _I'm not a fan of the double-indentation rustfmt introduces here... but I guess it doesn't really matter since these are just tests._

Reply:
I guess we could rewrite this to:

```rust
fn main() {
    unsafe { do_main() }
}

unsafe fn do_main() {
    let x = i32x2(1, 1);
    let y = i32x2(1, 0);
    simd_div(x, y); //~ERROR Undefined Behavior: dividing by zero
}
```

but it's not clear to me that's truly better.

Reply:
Personally I'd slap `rustfmt::skip` on this function :shrug: 
But this double-indentation is not the end of the world. We can leave it in here; if it becomes annoying in some other test we can always `skip` there.